.github/workflows/ruby.yml

```diff
@@ -26,3 +26,18 @@ jobs:
           bundler-cache: true
 
       - run: bundle exec rspec
+
+      - uses: actions/checkout@v3
```

Comment:
Is this line necessary? Looks like a spare.

Reply:
Thanks! That was a bad copy-paste on my part, fixed in the force-push to keep the commit clean.